### PR TITLE
Add K8S Network Section & Fix some typos

### PR DESCRIPTION
**Pull request by marianod92**
## What?
* Update K8S-EKS with Networking general considerations & examples
* Add reference to K8s-EKS Networking into main VPC Addressing Doc

## Why?
* To reflect in the documentation the latest updates of the reference architecture 

## References
* Related & Solved Issue: https://github.com/binbashar/le-tf-infra-aws/issues/185

### CODE REVIEW

**Review by exequielrafaela — COMMENTED** [review]
@marianod92 Thank you so much for your outstanding contributions to the binbash Leverage open source project. Your dedication and hard work are greatly appreciated by the entire community. Keep up the great work!

**Comment by marianod92** [comment]
image

**Review by diego-ojeda-binbash — APPROVED** [review]

**Review by angelofenoglio — APPROVED** [review]

File: docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    netmask and subnets with /24 netmasks. The primary reason AWS customers select smaller VPC and 
+    subnet sizes is to avoid overlapping network addresses with existing networks. 
+
+    So having [AWS single VPC Design](https://aws.amazon.com/answers/networking/aws-single-vpc-design/) we've chosen
+    a Medium/Small VPC/Subnet addressing plan which would probably fit a broad range variety of
+    use cases
+
+## Networking - IP Addressing
+
+!!! example "Starting CIDR Segment (AWS Org)"
```
Comment (exequielrafaela) [inline docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md:48]: @marianod92 I think here we should update this for a reference to the "EKS clusters", suggesting the commit below:

```suggestion
!!! example "Starting CIDR Segment (AWS EKS clusters)"
```

File: docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    * [x] AWS Org IP Addressing calculation is presented below based on segment `10.0.0.0/16` (starts at /16 due to AWS VPC limits)
```
Comment (exequielrafaela) [inline docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md:49]: @marianod92 here we should also refer to the "EKS clusters". Something similar to the suggestion below should work:

```suggestion
    * [x] AWS EKS clusters IP Addressing calculation is presented below based on segment `10.0.0.0/16` (starts at /16 due to AWS VPC limits)
```

File: docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    * [x] We started from `10.0.0.0/16` and subnetted to `/19`
+    * [x] Resulting in **Total Subnets: 8**
+      *   Number of available hosts for each subnet: 8190
+      *   Number of available IPs (AWS) for each subnet: 8187
+    
+!!! example "Individual CIDR Segments (VPCs)"
+    :fast_forward: Then each of these are /16 to /19
+    
+    *   [x] Considering the whole Starting CIDR Segment (AWS Org) before declared, we'll start at `10.0.0.0/16`
```
Comment (exequielrafaela) [inline docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md:58]: @marianod92 same than before comments referring to the clusters, possible suggestion below:

```suggestion
    *   [x] Considering the whole Starting CIDR Segment (AWS EKS clusters) before declared, we'll start at `10.0.0.0/16`
```

**Review by exequielrafaela — CHANGES_REQUESTED** [review]
@marianod92 This PR looks great, thanks for your contribution. I've just requested some minor editions. 

I would also like to request the possibility to add at the end of this section 👉🏼 https://leverage.binbash.com.ar/how-it-works/ref-architecture/ref-architecture-eks/ something similar to:

```
## Read More

!!! info "EKS Reference Architecture Specs" 
    In case you would like to further understand the different tech specs and configs for this Ref Arch you could find some details like at the [Features/Compute/K8s EKS section](../features/compute/k8s-eks/overview)
```

File: docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md (hunk 3 above)
Comment (marianod92) [reply docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md:58]: Done!

File: docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md (hunk 2 above)
Comment (marianod92) [reply docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md:49]: Done!

**Comment by marianod92** [comment]
> @marianod92 This PR looks great, thanks for your contribution. I've just requested some minor editions.
> 
> I would also like to request the possibility to add at the end of this section 👉🏼 https://leverage.binbash.com.ar/how-it-works/ref-architecture/ref-architecture-eks/ something similar to:
> 
> ```
> ## Read More
> 
> !!! info "EKS Reference Architecture Specs" 
>     In case you would like to further understand the different tech specs and configs for this Ref Arch you could find some details like at the [Features/Compute/K8s EKS section](../features/compute/k8s-eks/overview)
> ```

- Added Reference:

  image

File: docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md (hunk 1 above)
Comment (exequielrafaela) [reply docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md]: Thanks @marianod92!

File: docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md (hunk 3 above)
Comment (exequielrafaela) [reply docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md]: Great!

File: docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md (hunk 2 above)
Comment (exequielrafaela) [reply docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md]: Thanks! 🚀

**Pull request merged**